odoo/openupgrade/doc/source/modules90-100.rst

```diff
@@ -457,15 +457,15 @@ Status :
 +-----------------------------------+-----------------------------------+
 |web_analytics                      |                                   |
 +-----------------------------------+-----------------------------------+
-|web_calendar                       |                                   |
+|web_calendar                       | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|web_diagram                        |                                   |
+|web_diagram                        | Nothing to do                     |
```

Comment:
As this module contains a migrations folder, it's convenient to include the openupgrade_analysis_work.txt file indicating that there's nothing to do. This way, looking at the folder structure without checking this documentation, you can see if the module has been analyzed.